### PR TITLE
fix(specs): Personalization API

### PR DESCRIPTION
**Pull request by kai687**
## 🧭 What and Why

Update the descriptions of the Personalization API to improve the reference content.

### CODE REVIEW

File: specs/personalization/common/schemas/eventScoring.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ eventScoring:
       $ref: 'personalizationStrategy.yml#/score'
     eventName:
       type: string
-      description: The name of the event.
+      description: Event name.
     eventType:
       type: string
-      description: The type of the event.
+      description: Event type (click, conversion, or view).
```
Comment (millotp) [inline specs/personalization/common/schemas/eventScoring.yml:16]: could this be an enum ?

**Review by millotp — DISMISSED** [review]
thanks !

**Review by millotp — APPROVED** [review]
all good !

**Review by shortcuts — APPROVED** [review]
ggggggg

**Pull request merged**